OpenOversight/app/models.py

```diff
@@ -59,6 +60,24 @@ def toCustomDict(self):
             "unique_internal_identifier_label": self.unique_internal_identifier_label,
         }
 
+    def latest_incident_date(self) -> datetime.date:
```

Comment:
Should we cache these at all? Would that make sense to do/affect anything?

Reply:
Yup, that makes a lot of sense for these values! Do you have any suggestions on what to use for caching? I know there's the built-in functools cache but we probably want to be able to set a TTL on these dates

Reply:
Added cachetools caching!